pgx/src/datum/array.rs

```diff
@@ -73,48 +100,82 @@ impl<'a, T: FromDatum> Array<'a, T> {
         // Remember to remove the Array::over tests in pgx-tests/src/tests/array_tests.rs
```

Comment:
Does Rust have something similar to a `TODO` that can report on things like "hey check this out later"?

Reply:
https://docs.rs/todo-or-die/latest/todo_or_die/ ?

Reply:
Harsh crate name 😆 

I was thinking maybe a one-off cargo command or something that will parse the codebase and say "you have these todos, check them out". Though, in saying it out loud in my head, that's honestly no different than a `rg TODO`